pandas/tests/plotting/test_style.py

```diff
@@ -1,6 +1,11 @@
+import numpy as np
```

Comment:
why did you move here? these should all be with the styler tests.

Reply:
very obvious in hindsight what you intended.